### ISSUE TITLE
API: handle feature presets

### ISSUE THREAD

**Issue by f4exb** [f4exb/sdrangel#998]
In order to further complete the handling of features via API it is also necessary to support feature presets. This is more or less based on the existing endpoints for presets. Presently PUT, PATCH, POST to `/sdrangel/preset` is not handled quite properly since it applies to a particular deviceset it should be moved to the deviceSet section and have `/sdrangel/deviceset/{deviceSetIndex}/preset` as endpoint. This may be covered in another issue later.

So to summarize what has to be added:
  - in Instance section:
    - GET `/sdrangel/featurepresets`
    - DELETE `/sdrangel/featurepreset`
  - In FeatureSet section:
    - PUT, PATCH, POST to `/sdrangel/featureset/{featureSetIndex}/preset` 

Note: this is a prerequisite to https://github.com/f4exb/sdrangelcli/issues/38

**Comment by f4exb** [comment]
Implemented in v6.16.5
